comp/trace/bundle_test.go

```diff
@@ -39,6 +41,7 @@ func TestBundleDependencies(t *testing.T) {
 		statsd.Module(),
 		fx.Provide(func(cfg config.Component) telemetry.TelemetryCollector { return telemetry.NewCollector(cfg.Object()) }),
 		fx.Supply(tagger.NewFakeTaggerParams()),
+		fx.Provide(func(cfg config.Component) compression.Component { return gzip.NewComponent() }),
```

Comment:
We can remove the `cfg config.Component` from the function arguments as we are not using it to decide the compression component 😄